app/services/repository_service.rb

```diff
@@ -0,0 +1,60 @@
+class RepositoryService
+
+  class RepositoryNotFound < RuntimeError
+  end
+
+  def create_repository!(product, url, attributes, custom: false)
+    repository = Repository.find_or_initialize_by(external_url: url)
+
+    # TODO: See if we can clean this up
```

Comment:
Please clarify the problem here.

Reply:
This was extracted from older code during the refactoring process, nothing has changed, but I don't like it at all. 

Unfortunately, I haven't see an easy way to clean it up, so I just made a note to clean it up in the future. Also, this was from a previous commit, but the file was renamed so it shows a change when there really is none.

Reply:
@tmuntan1 Let's remove the TODO then. No point in keeping it in the code forever.